plugins/dell-dock/fu-dell-dock-common.c

```diff
@@ -43,3 +43,20 @@ fu_dell_dock_set_power(FuDevice *device, guint8 target, gboolean enabled, GError
 
 	return fu_dell_dock_ec_modify_lock(parent, target, enabled, error);
 }
+
+const gchar *
+fu_dell_dock_get_instance_id(guint8 *type, DellDockComponent *dev_list, guint16 *vid, guint16 *pid)
```

Comment:
type, vid and pid are pointers to integers, but why?  You don't seem to update or use them as "out" values.  I think they can all just be regular arguments.

Reply:
Got it, fixed in [8f86ae8](https://github.com/fwupd/fwupd/commit/8f86ae85077835c77fcf446e55a2632f3d8ac61b)